docker/docker-compose.yml

```diff
@@ -0,0 +1,30 @@
+concourse-db:
```

Comment:
Can we pin all (postgres and concourse) container image versions?
We want reproducible deployments.